### PR TITLE
Committbranch

### PR DESCRIPTION
**Pull request by carlos-molina**

### CODE REVIEW

**Comment by carlos-molina** [comment]
I managed to revert the commits I included in the first Pull Request. Thus only commit 3e8f8f3 is active. 

The commit  is related to four files: 

1) slides/fosdem14/index.html and slides/oscon14/content.md have been altered to include the UCN project logo and a "Thanks to UNC project" line.  

2) ucn_logo.png has been included as slides/oscon14/ucn_logo.png  and as slides/fosdem14/ucn_logo.png.

**Comment by carlos-molina** [comment]
I managed to revert the commits I included in the first Pull Request. Thus only commit 3e8f8f3 is active.

The commit  is related to four files:

1) slides/fosdem14/index.html and slides/oscon14/content.md have been altered to include the UCN project logo and a "Thanks to UNC project" line.

2) ucn_logo.png has been included as slides/oscon14/ucn_logo.png  and as slides/fosdem14/ucn_logo.png.

File: slides/fosdem14/index.html [hunk 1]
```diff
@@ -784,6 +786,11 @@
          + **<https://github.com/mirage>**
          + **<https://github.com/xapi-project>**
          + **<https://github.com/xenserver>**
+         + **<https://github.com/xenserver>**
```
Comment (mor1) [inline slides/fosdem14/index.html]: duplicated line?

**Comment by mor1** [comment]
this needs to be done today; i'll copy the relevant patches from this PR and apply them. thanks.

**Pull request closed without merging**